README.md

```diff
@@ -35,6 +35,7 @@ There are two types of tutorials,
 ### Kubernetes
 
 - [What is kubernetes](https://deis.com/blog/2016/kubernetes-illustrated-guide/) `explaination`
+- [Minikube](https://kubernetes.io/docs/setup/minikube/) `code` `explaination`
```

Comment:
The title should be something easy to know about the source. In your case, **Minikube official docs**.
Simply Minikube is an ambiguous title. 
Change the title and then it's good to go :).

Reply:
Thanks, I changed the title.